test/unit/sequence_base_test.rb

```diff
@@ -58,6 +58,32 @@ class SequenceBaseTest < MiniTest::Test
         end
       end
     end
+
+    it 'passes when string value starts with searched value' do
```

Comment:
Should these tests be in the `US Core Patient Test`? Or is that the wrong place for them (either because they're autogenerated, or another reason?)

Reply:
I agree that it is very odd to have a dependency on USCore311 tests here, and ideally the code would be written such that you could wouldn't need a dependency on a specific test.  What if we remove the test.  Having said that, all tests in this entire file have this problem, the addition adds useful test coverage, and we're planning on migrating off this platform anyhow in the future... so a big refactor to fix this doesn't make sense to me.